models/public/midasnet/model.yml

```diff
@@ -0,0 +1,55 @@
+# Copyright (c) 2019 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+description: >-
+  MidasNet is a model for monocular depth estimation trained by mixing several datasets;
+  as described in the following paper: 
+  "Towards Robust Monocular Depth Estimation: Mixing Datasets for Zero-Shot Cross-Dataset Transfer"
+  <https://arxiv.org/pdf/1907.01341>
+
+  The model input is a blob that consists of a single image of "1x3x384x384" in RGB order.
+
+  The model output is an inverse depth map that is defined up to an unknown scale factor.
+task_type: mono_depth
+files:
+  - name: midasnet.py
+    size: 6464
+    sha256: a3680912f1e6839b5bc4d4684e78b2a8f5f669125ee86aff81b53dbe8d635a12
+    source:
+      $type: google_drive
+      id: 1fL_UDsZ8pFUN3QnXqMF4BReVZmfmrKBj
+  - name: midasnet.pth
+    size: 422388453
+    sha256: f46da7438f3dd46a46fa550cea1d5dcab4bb3192ba272f1fd2b08d23ba41d39e
+    source:
+      $type: google_drive
+      id: 1m6-zyyiZj2XCiemSdQ7ldrEW-E1ZFfIl
```

Comment:
This appears to be the same file that is linked from the repository, but it's been reuploaded under a different ID. Why reupload it?

Reply:
See my comment below: We needed to apply changes to our original code to make the model convertible to dltl (via onnx). We want to keep the weights and (adapted) model definition consistent. That is why we keep them in the same gdrive folder.